Library/Homebrew/cask/dsl.rb

```diff
@@ -106,7 +106,7 @@ class DSL
     ]).freeze
 
     extend Attrable
-    include OnSystem::MacOSOnly
+    include OnSystem::MacOSAndLinux
```

Comment:
FYI: I think this is the last reference to `OnSystem::MacOSOnly` in the codebase so the definition and associated types can probably be removed too.

Reply:
If that's okay with you I'd like to postpone that until this PR is merged at least.